### PR TITLE
CleverTap Android Kit - Initial

### PR DESCRIPTION
**Pull request by pwilkniss**

### CODE REVIEW

File: src/main/java/com/mparticle/kits/CleverTapKit.java [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+        if (intent == null || intent.getExtras() == null) {
+            return false;
+        }
+        NotificationInfo info = CleverTapAPI.getNotificationInfo(intent.getExtras());
+        return info.fromCleverTap;
+    }
+
+    @Override
+    public void onPushMessageReceived(Context context, Intent pushIntent) {
+        if (pushIntent == null || pushIntent.getExtras() == null) {
```
[the remark this answers is not in the document]
Comment (pwilkniss) [reply src/main/java/com/mparticle/kits/CleverTapKit.java]: thanks!  so just to be clear we can assume its our push in onPushMessageReceived, so can reduce to this?

 @Override
    public void onPushMessageReceived(Context context, Intent pushIntent) {
        if (pushIntent == null || pushIntent.getExtras() == null) {
            return;
        }
        Bundle extras = pushIntent.getExtras();
        CleverTapAPI.createNotification(getContext(), extras);
    }

**Comment by pwilkniss** [comment]
thanks for the review Sam, committed the two suggestions (duh :)).  just a clarification on the other comment and then can commit that as well.

File: src/main/java/com/mparticle/kits/CleverTapKit.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (samdozor) [reply src/main/java/com/mparticle/kits/CleverTapKit.java]: yep exactly. or just send it into your broadcastreceiver if you have one and let it handle it like anything else? up to you.

[the remark this answers is not in the document]
Comment (pwilkniss) [reply src/main/java/com/mparticle/kits/CleverTapKit.java]: ok, that's pushed.  thanks!

**Comment by alexs-mparticle** [comment]
@pwilkniss looks like you have a merge conflict. Would you be able to resolve this?

**Comment by darshanclevertap** [comment]
@alexs-mparticle Stepping in for @pwilkniss 
This PR is very old and will need updates.
Give me a day to update and resolve conflicts as well.
Thanks!

**Comment by darshanclevertap** [comment]
@alexs-mparticle on second look, this PR can be closed. This PR is from our [old fork](https://github.com/CleverTap/mparticle-android-integration-clevertap_old). 
CleverTap integration with mParticle is live from our [new fork here](https://github.com/CleverTap/mparticle-android-integration-clevertap)

**Comment by alexs-mparticle** [comment]
@darshanclevertap Thanks for the heads up and helping us dust off some of these old issues.

**Pull request closed without merging**